### PR TITLE
Update quality-gates docs

### PR DESCRIPTION
**Pull request by liashenko**
### What does this PR do?
* Update quality-gates docs to suggest the new version of `datadog-ci gate` 
* Add `--timeout` flag description
* Update `DATADOG_APP_KEY` and `DATADOG_API_KEY` to `DD_APP_KEY` and `DD_API_KEY`

### Motivation
* The new version of `datadog-ci gate` contains improved wait time handling, and a new `timeout` flag introduced in https://github.com/DataDog/datadog-ci/pull/1013
* We want customers to use `DD_APP_KEY` and `DD_API_KEY`

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/continuous_integration/quality_gates/_index.md [hunk 1]
```diff
@@ -81,6 +81,7 @@ The behavior of the command can be modified using the following flags:
 based on the current pipeline context. By default, the command succeeds.
 - `--fail-if-unavailable`: when this flag is specified, the command fails if one or more rules could not be evaluated because of an internal issue.
 By default, the command succeeds.
+- `--timeout`: the command will stop its execution after the specified timeout in seconds. The default timeout is 10 minutes.
```
Comment (alxch-) [inline content/en/continuous_integration/quality_gates/_index.md:84]: Can you describe cases where it can happen?

Comment (liashenko) [reply content/en/continuous_integration/quality_gates/_index.md]: I've updated the description, wdyt?

File: content/en/continuous_integration/quality_gates/_index.md [hunk 2]
```diff
@@ -51,7 +51,7 @@ The following example shows how to create a static analysis rule that fails when
 
 ## Invoking quality gates
 
-To use quality gates, your [`datadog-ci`][7] version should be higher or equal than `2.16.0`.
+To use quality gates, your [`datadog-ci`][7] version should be higher or equal than `2.19.0`.
```
Comment (hestonhoffman) [inline content/en/continuous_integration/quality_gates/_index.md:54]:
```suggestion
To use quality gates, your [`datadog-ci`][7] version should be `2.19.0` or later.
```

**Review by hestonhoffman — COMMENTED** [review]
Small edit

**Review by hestonhoffman — APPROVED** [review]

**Review by alxch- — APPROVED** [review]

**Pull request merged**